learn/advanced/updating.md

```diff
@@ -225,35 +176,192 @@ Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move
+```
+
+:::
+
+::::
 
-Now that you’ve got your dump, install the [latest version of Meilisearch](/learn/getting_started/quick_start.md#setup-and-installation) and [import the dump](/learn/advanced/dumps.md#importing-a-dump) at launch using the CLI option:
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For cloud platforms, move the new Meilisearch binary to the `/usr/bin` directory containing the executable files:
+
+```
+mv meilisearch /usr/bin/meilisearch
+```
+
+## Step 7: Launch Meilisearch and import the dump
+
+Execute the command below to import the dump at launch:
+
+:::: tabs
+
+::: tab Local installation
 
 ```bash
-# launch the latest version of Meilisearch with the master key and import the specified dump file
-./meilisearch --import-dump /dumps/your_dump_file.dump --master-key="MASTER_KEY"
+# replace {dump_uid.dump} with the name of your dump file
+./meilisearch --import-dump /dumps/{your_dump_file.dump} --master-key="MASTER_KEY"
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {dump_uid.dump} with the name of your dump file
+meilisearch --db-path /var/lib/meilisearch/data.ms --import-dump "/var/opt/meilisearch/dumps/{dump_uid.dump}"
 ```
 
+:::
+
+::::
+
 ::: warning
 If you are using Meilisearch v0.20 or below, migration should be done in two steps. First, import your dump into an instance running any version of Meilisearch from v0.21 to v0.24, inclusive. Second, export another dump from this instance and import it to a final instance running your targeted version.
+
+Once Meilisearch v1 is released, this two-step process won't be necessary as v1 will be compatible with dumps from all previous versions.
 :::
 
 Importing a dump requires indexing all the documents it contains. Depending on the size of your dataset, this process can take a long time and cause a spike in memory usage.
 
-Finally, don’t forget to set `displayedAttributes` back to its previous value if necessary. You can do this using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+::: danger `_geo` field in v0.27, v0.28, and v0.29
+You might not be able to import your dump due to Meilisearch allowing malformed `_geo` fields in the above-mentioned versions. Please ensure the `_geo` field follows the [correct format](/learn/advanced/geosearch.md#preparing-documents-for-location-based-search).
+:::
+
+## Step 8: Restart Meilisearch as a service
+
+Once your dump has been correctly imported, press `Ctrl`+`C`  to stop Meilisearch. Next, execute the following command to run the script to configure Meilisearch and restart it as a service:
+
+```
+meilisearch-setup
+```
+
+If required, set `displayedAttributes` back to its previous value using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+
+## Step 9: Delete backup files or rollback
```

Comment:
The content of the step looks good but the order is a bit off. Process should be:

- Verify that dump import was successful and no data or settings were lost
- If data was lost, rollback
- If data was not lost, delete backups

Instructing people to delete their backups (which is not reversible btw) then saying "use the files you just deleted to rollback" seems like a bad idea.